liota/utilities/protocol_utility.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
```

Comment:
Leverage paho provided error code and conn ack code lookup methods. We should not attempt to re-map code and description once again. Refer https://github.com/eclipse/paho.mqtt.python/blob/master/src/paho/mqtt/client.py